README.md

```diff
@@ -22,12 +22,27 @@ docker run -d
   -e DB_DATABASE='ninja'
   -e DB_USERNAME='ninja'
   -e DB_PASSWORD='ninja'
+  -e PHANTOMJS_SECRET=`gVBUWzLjRWQq0lLqiO5RnGHvugUn+OuszNvymIiGex4jtrqznQa278sxHNNZZuhls54FhXWs9wFqN2pYmy8zKHEsMJoi60CSMZ6hkMiOymCf1IbHr2SxmidXeYyW+ZO6QjW6T2SwiJQH2fUFN82/yxtKHaDMz+7ilNQjZ0RGCQI=`
```

Comment:
@jbzdak sorry for the long silence, as @hillelcoren mentined, can you please remove hard coded key and replace this by an explicit string ?

Reply:
Also, shouldn't these be enclosed in single quotes, and not backticks?